prometheus-client/prometheus-client-config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "modelPackage": "org.candlepin.subscriptions.prometheus.model",
+  "apiPackage": "org.candlepin.subscriptions.prometheus.resources",
+  "invokerPackage": "org.candlepin.subscriptions.prometheus",
+  "groupId": "org.candlepin",
+  "artifactId": "prometheus-client",
+  "java8": "true"
+}
```

Comment:
Nitpick: newline missing

Reply:
don't need the "java8":"true" anymore, right?

Reply:
DONE - Available in next commit.